pkg/cmd/start/main.go

```diff
@@ -18,11 +18,11 @@ import (
 func AddFlags(cmd *cobra.Command) {
 	cmd.Flags().String("log-level", "info", "The log-level for the operator. Possible values: trace, debug, info, warning, error, fatal, panic")
 	cmd.Flags().String("jaeger-version", version.DefaultJaeger(), "Deprecated: the Jaeger version is now managed entirely by the operator. This option is currently no-op.")
-	cmd.Flags().String("jaeger-agent-image", "jaegertracing/jaeger-agent", "The Docker image for the Jaeger Agent")
-	cmd.Flags().String("jaeger-query-image", "jaegertracing/jaeger-query", "The Docker image for the Jaeger Query")
-	cmd.Flags().String("jaeger-collector-image", "jaegertracing/jaeger-collector", "The Docker image for the Jaeger Collector")
-	cmd.Flags().String("jaeger-ingester-image", "jaegertracing/jaeger-ingester", "The Docker image for the Jaeger Ingester")
-	cmd.Flags().String("jaeger-all-in-one-image", "jaegertracing/all-in-one", "The Docker image for the Jaeger all-in-one")
+	cmd.Flags().String("jaeger-agent-image", "quay.io/maistra/jaeger-agent-rhel7:1.17.6-ibm", "The Quay image for the Jaeger Agent")
```

Comment:
Can't you just pass the flags instead? You can change the following:

https://github.com/jaegertracing/jaeger-operator/blob/729d8aaaa08aada07a790a900a18867dbd98deb1/deploy/operator.yaml#L24

To use:

```yaml
        args: ["start", "--jaeger-all-in-one-image=quay.io/maistra/jaeger-all-in-one-rhel7:1.17.6-ibm"]
```

This should be applied to your local `operator.yaml`. This change really doesn't belong to this repo here.

Reply:
@madeelrh when you install jaeger-operator from OperatorHub these changes will be included. No actions required here.